docs/quick_start.md

```diff
@@ -107,7 +106,7 @@ const FAUCET_KEY = {
     'setup',
     'rescue',
   ],
-  secret: '35f266fbf0fca752da1342fdfc745a9c608e7b20',
+  activation_code: '35f266fbf0fca752da1342fdfc745a9c608e7b20',
```

Comment:
At line 121, FAUCET_KEY.secret needs to be replaced with FAUCET_KEY.activation_code

Reply:
oh! missed that. Thanks! Fixe.